buildenv/jenkins/common/build.groovy

```diff
@@ -406,6 +426,14 @@ def archive_sdk() {
                         echo "Javadoc (OpenJ9 extensions only):'${JAVADOC_OPENJ9_ONLY_LIB_URL}'"
                     }
                 }
+                if (params.CODE_COVERAGE) {
+                    if (fileExists("${CODE_COVERAGE_FILENAME}")) {
+                        CODE_COVERAGE_LIB_URL = "${ARTIFACTORY_CONFIG[ARTIFACTORY_CONFIG['defaultGeo']]['url']}/${ARTIFACTORY_CONFIG['uploadDir']}${CODE_COVERAGE_FILENAME}"
+                        currentBuild.description += "<br><a href=${CODE_COVERAGE_LIB_URL}>${CODE_COVERAGE_FILENAME}</a>"
```

Comment:
I expect quotes are required around `${CODE_COVERAGE_LIB_URL}` here (to yield legal html).

Reply:
Yes, but the generated html needs to have quotes around the URL:
```
<br><a href="Internal_Jenkins/job/Build_JDK11_aarch64_linux_Personal/176/">code-coverage-files.tar.gz</a>
```
Are those quotes included in the value of `${CODE_COVERAGE_LIB_URL}`?

Reply:
Quotation has been added for ${CODE_COVERAGE_LIB_URL} as suggested. Thanks.